autoscheduler/scraper/management/commands/utils/scraper_utils.py

```diff
@@ -35,15 +39,15 @@ def get_recent_semesters(now=datetime.now()) -> List[str]:
     spring_reg_start = datetime.strptime(f'10/26/{year}', date_format)
 
     # For the comments, use now.year = 2020
-    # Between [01/01/2020, 04/01/2020)
+    # Between [01/01/2020, 03/22/2020)
     if now < summer_fall_reg_start:
         return [f"{year}1"]
 
-    # Between [04/01/2020, 11/05/2020]
+    # Between [03/22/2020, 10/26/2020]
     if summer_fall_reg_start <= now < spring_reg_start:
         return [f"{year}2", f"{year}3"]
 
-    # Between [11/05/2020, 04/01/2021]
+    # Between [10/26/2020, 03/22/2021]

```

Comment:
```suggestion
    # Between [10/26/2020, 12/31/2020]
```
Isn't this right?

Reply:
Make sure you address this comment

Reply:
No, look at the dates above them - I shifted them back a few days so that the scraper would have enough time to scrape them (I think the data was available ~9 days before registration opened)